PythonInfoSystem/contents/templates/contents/forms.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.models import User
```

Comment:
This file should be located in the main app's directory, not in templates folder.